src/targets.sh

```diff
@@ -79,6 +80,24 @@ elif [[ "$1" == "attributes" ]]; then
     "$DIR/get" "/targets/$2/attributes" | \
         jq --raw-output '. | to_entries[] | [ .key, .value ] | @tsv' | \
         column -s '	' -t
+elif [[ "$1" == assignDS ]]; then
```

Comment:
a bit of a nit-pick, but everywhere else we compare to "string literals" (I think thats what they are called?):

```bash
elif [[ "$1" == "assignDS" ]]; then
```